pl.js

```diff
@@ -0,0 +1,688 @@
+settings.mods.overlay.paste URL obrazu nakładki gry
+settings.social.region Statystyki regionu
+windows.settings.header Ustawienia gry
+windows.mail.header Krunker Mail
+windows.servers.header Serwery
+windows.servers.none Nie znaleziono dołączalnych gier...
+windows.servers.host + Hostuj grę
+windows.servers.hide Ukryj pełne
+windows.servers.priority Priorytet regionu
+windows.servers.online {0} W grze
+windows.servers.social {0} w Social
+windows.loadout.header Zmień wyposażenie
+windows.loadout.class Klasa
+windows.loadout.primary Broń główna
+windows.loadout.primary.attach Dodatek
+windows.loadout.primary.reticle Celownik
+windows.loadout.primary.scope Luneta
+windows.loadout.secondary Druga broń
+windows.loadout.secondary.skin Skin drugiej broni
+windows.loadout.melee Broń biała
+windows.loadout.hat Kapelusz
+windows.loadout.body Ciało
+windows.loadout.waist Talia
+windows.loadout.shoe Buty
+windows.loadout.face Twarz
+windows.loadout.spray Sprej
+windows.loadout.color.skin Kolor skóry
+windows.loadout.color.chat Kolor nazwy w czacie
+windows.loadout.color.hair Kolor włosów
+windows.loadout.dye Barwnik
+windows.loadout.pet Zwierzak
+windows.loadout.stat Statystyki przy zabiciu
+windows.loadout.flag Flaga kraju
+windows.mods.drop upuść tu plik modyfikacji lub naciśnij
+windows.mods.paste Wklej URL modyfikacji
+windows.mods.load Załaduj modyfikacje
+windows.mods.none Nie znaleziono modyfikacji
+windows.mods.header Menedżer modyfikacji
+windows.mods.guide Poradnik tworzenia modyfikacji
+windows.mods.list Lista modyfikacji
+windows.mods.load.header Załaduj modyfikacje
+windows.mods.publish Opublikuj modyfikacje
+windows.mods.viewer Widz
+windows.account.header Konto
+windows.account.recover Aby przywrócic konto, skontaktuj się z <a>krunker@yendis.ch</a>
+windows.class.header Wybierz klasę
+windows.controls.header Zmień ustawienia kontroli
+windows.controls.forward Naprzód
+windows.controls.backward Do tyłu
+windows.controls.left Lewo
+windows.controls.right Prawo
+windows.controls.reload Przeładuj
+windows.controls.aim Celuj
+windows.controls.shoot Strzelaj
+windows.controls.inspect Pokaż broń
+windows.controls.spray Sprej
+windows.controls.jump Skakanie
+windows.controls.crouch Przyklęknięcie
+windows.controls.prim Główna broń
+windows.controls.swap Druga broń
+windows.controls.melee Broń biała
+windows.controls.equip Wyposażenie
+windows.controls.chat Czat
+windows.controls.voice Głos
+windows.controls.list Lista graczy
+windows.controls.board Tabela wyników
+windows.controls.interact Działanie
+windows.controls.interactSec Drugie działanie
+windows.controls.confirm Potwierdź działanie
+windows.controls.drop Upuść
+windows.controls.wepVis Przełącz widoczność broni
+windows.controls.streak0 Seria 1
+windows.controls.streak1 Seria 2
+windows.controls.streak2 Seria 3
+windows.controls.streak3 Seria 4
+windows.controls.streak4 Seria 5
+windows.controls.premium0 Noclip
+windows.controls.premium1 Tryb boga
+windows.controls.premium2 Zabicie
```

Comment:
I think this one should be changed to "Zabij"

Reply:
ok